assets/js/javascript.js

```diff
@@ -136,12 +144,14 @@ function apiCall(baseUrl, params = {}) {
   return fetch(requestUrl).then( function(response) {
     if (!response.status == 200) {
       // TODO: 404 Redirect
+      alert('Spoonacular responded with status 200');
```

Comment:
This alert should display response.status as a variable, this only displays if the response.status was NOT 200 (not successful).